src/controller/index.ts

```diff
@@ -133,13 +139,21 @@ export default class SimulariumController {
                 this.visData.parseAgentsFromLocalFileData.bind(this.visData)
             );
         } else if (netConnectionConfig) {
-            this.simulator = new RemoteSimulator(
+            const webSocketClient = new WebsocketClient(
                 netConnectionConfig,
                 this.onError
             );
+            this.simulator = new RemoteSimulator(webSocketClient, this.onError);
             this.simulator.setTrajectoryDataHandler(
                 this.visData.parseAgentsFromNetData.bind(this.visData)
             );
+            this.converter = new RemoteConverter(webSocketClient, this.onError);
+            this.converter.setLoadFileHandler((result: ConvertedFileData) => {
+                const file = JSON.parse(result["simulariumData"]);
+                const simulariumFile = new JsonFileReader(file);
+                // TODO: make file name more informative
+                this.handleFileChange(simulariumFile, "test.simularium");
+            });
```

Comment:
I get why you're initializing this here, but it also seems like it could be done separately, and only if the user requests a conversion? 
Otherwise, I think it would be good to separate out this section contained in the `else if` block and call it `setUpRemoteConnection` or something, because this goes beyond `createSimulatorConnection`